src/mono/wasm/runtime/loader/blazor/WebAssemblyResourceLoader.ts

```diff
@@ -33,7 +34,15 @@ export class WebAssemblyResourceLoader {
             ? this.loadResourceWithCaching(this.cacheIfUsed, name, url, contentHash, resourceType)
             : this.loadResourceWithoutCaching(name, url, contentHash, resourceType);
 
-        return { name, url: toAbsoluteUri(url), response };
+        const absoluteUrl = toAbsoluteUri(url);
+
+        if (resourceType == "assembly") {
+            if (!loaderHelpers.loadedAssemblies)
```

Comment:
you already initialized that in globals.ts right?

Reply:
I'll check it once again, I sow failures about being it undefined